packages/gatsby-plugin-google-analytics/src/index.js

```diff
@@ -62,6 +62,9 @@ function OutboundLink(props) {
 OutboundLink.propTypes = {
   href: PropTypes.string,
   target: PropTypes.string,
+  eventCategory: PropTypes.string,
+  eventAction: PropTypes.string,
+  eventLabel: PropTypes.string,
```

Comment:
Adding those props will add non-standard HTML attributes to the `a` tag above:

https://github.com/gatsbyjs/gatsby/blob/e459c133368ea119d337f78876a334ad722af2dc/packages/gatsby-plugin-google-analytics/src/index.js#L18-L20

I guess you should destructure those props and pass only the rest of them to the `<a>` tag.